src/Core/pizza.h

```diff
@@ -0,0 +1,38 @@
+/*
+** EPITECH PROJECT, 2022
+** plazza
+** File description:
+** pizza
+*/
+
+/// \file src/Core/pizza.h
+
+#ifndef PIZZA_H_
+    #define PIZZA_H_
+
+    #include <string>
+
+namespace pizza {
+    typedef pizzaSize_e pizzaSize_t;
```

Comment:
Why the typedef ?

Reply:
because

Reply:
Usefull, i don't want to use "enum pizzaSize_e" in the entire project, I prefer "pizzaSize_t"